client/components/common/Footer.vue

```diff
@@ -1,19 +1,15 @@
 <template>
   <v-footer height="46" color="primary darken-4" absolute>
-    <v-row justify="center" no-gutters>
-      <v-col
-        cols="12"
-        class="body-2 grey--text text--lighten-4">
-        <img :src="logo" width="30px" height="30px" alt="Logo" class="py-1 mx-1">
-        <v-chip
-          color="grey lighten-4"
-          label small
-          class="readonly mr-3 grey--text text--darken-4">
-          v{{ version }} {{ codename }}
-        </v-chip>
-        Built with <v-icon color="pink">mdi-heart</v-icon>
-        Extension Engine
-      </v-col>
+    <v-row align="center" justify="center" class="body-2 grey--text text--lighten-4">
```

Comment:
Consider replacing with div and align/justify classes since we removed the [col](https://vuetifyjs.com/en/components/grids/#v-row) 😉

Reply:
Addressed in 00980320964b0a06eb2c70d24c5f84d37fba799f

Reply:
Accidental c/p fixed in 36c04c4c63fad725d05b3631c325b18972069733.